protos/OBF.proto

```diff
@@ -609,6 +609,7 @@ message OsmAndPoiBoxDataAtom {
 
   repeated uint32 textCategories = 14; // v1.7
   repeated string textValues = 15;
+  optional int32 precisionXY = 16; // precision in 1-xy-xy-xy binary format
```

Comment:
This field is variable-length precision-improvement from 24th zoom up to X zoom. It's encoded by interleaving next zoom level bits with a high bit prefix of "1". So, precisionXY from 24 -> 25 zoom could be 3 any bit number: 110, 111, 100,  101. From 24 -> 26 - 5 bit number, etc...